cleanlab/token_classification/rank.py

```diff
@@ -0,0 +1,112 @@
+import pandas as pd
+import numpy as np
+from cleanlab.rank import get_label_quality_scores as main_get_label_quality_scores
+
+
+def get_label_quality_scores(
+    labels: list,
+    pred_probs: list,
+    *,
+    tokens: list = None,
+    token_scoring_method: str = "self_confidence",
+    sentence_scoring_method: str = "min",
+    param: float = 0.04,
+    return_scores_per_token: bool = True,
+):
+    """Returns label quality scores for each sentence
```

Comment:
"Returns label quality scores for each sentence" 
-> 
Returns overall quality scores for the labels in each sentence (as well as for the individual tokens' labels)